otter/models/cass.py

```diff
@@ -1318,12 +1318,12 @@ def _delete_lock_znode(result):
                     exc=f.value,
                     otter_msg_type="ignore-delete-lock-error"))
 
-        lock = self.kz_client.Lock(LOCK_PATH + '/' + self.uuid)
-        lock.acquire = functools.partial(lock.acquire, timeout=120)
+        lock = zk.PollingLock(self.dispatcher, LOCK_PATH + '/' + self.uuid)
+        lock.acquire = functools.partial(lock.acquire, timeout=10)
```

Comment:
> why is it going down by an order of magnitude?

Good question. Actually, the diff is not up-to-date. It should not show 120. I changed it from 120 to 10 in #1923 mainly to see if `LockTimeouts` reduce. 120 was not good because 1 orphan child node will keep many pending requests running for 120 seconds. They will all be timed out by CLB in 30 seconds but unnecessarily remain in the system. When analyzing sample requests I found that the time the lock is actually held is for 10-15 ms. Hence, I wanted to change it to some number < 30s. I'll add some documentation and make this a constant. It might be complicated to make this configurable for now.